containers/rust-hello-world/src/main.rs

```diff
@@ -0,0 +1,31 @@
+use std::{
+    io::{prelude::*, BufReader},
+    net::{TcpListener, TcpStream},
+};
+
+fn main() {
+    let listener = TcpListener::bind("127.0.0.1:8080").unwrap();
+
+    for stream in listener.incoming() {
+        let stream = stream.unwrap();
+
+        handle_connection(stream);
+    }
+}
+
+fn handle_connection(mut stream: TcpStream) {
+    let buf_reader = BufReader::new(&mut stream);
+    let _http_request: Vec<_> = buf_reader
```

Comment:
question: can we do something with this variable? It's great to have it to show users how to parse the HTTP request, but here we are not doing anything with it. I'd say: either we add a comment saying "this is how we parse HTTP request" OR (better IMO) we use it in the code.

Maybe something like: calling `/` without arguments sends "hello world", but calling it with `?name=<something>` will return "hello <something>"? WDYT?

Reply:
added a simple comment, the example is not about rust itstelf 👌